src/Module/Contact.php

```diff
@@ -140,25 +140,25 @@ private static function batchActions(App $a)
 		$count_actions = 0;
 		foreach ($orig_records as $orig_record) {
 			$contact_id = $orig_record['id'];
-			if (defaults($_POST, 'contacts_batch_update')) {
+			if (defaults($_POST, 'contacts_batch_update', '')) {
```

Comment:
Why did you add this default value?

Reply:
You're right, when used with two parameters, the signature is `defaults($var, $default)`.